src/components/pages/Fuse/FuseReferralBar.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button } from "@chakra-ui/react";
+import { RowOrColumn, Center } from "buttered-chakra";
+import React from "react";
+import { useParams } from "react-router-dom";
+import { useIsSmallScreen } from "../../../hooks/useIsSmallScreen";
+import DashboardBox from "../../shared/DashboardBox";
+import { mediumAddress } from "../../../utils/shortAddress";
+import { useRari } from "../../../context/RariContext";
+
+const FuseReferralBar = () => {
+    const isMobile = useIsSmallScreen();
+    let { poolId } = useParams();
+    const { address } = useRari();
+
+    const short_referral_code = `https://app.rari.capital/fuse/pool/${poolId}/${mediumAddress(address)}`;
```

Comment:
can we make this relative or get the URL programmatically?

Reply:
yup, gotcha